### PR TITLE
Feature/billy add property button 2

### PR DESCRIPTION
**Pull request by billyhunt**
Add property button and other UI elements.  

image

### CODE REVIEW

File: frontend/src/api/addProperty.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  console.log("search query", query)
+  return axios
+    .post(`${apiEndpoint}/property/search`, query)
+    .then((r) => {
+      return Promise.resolve(r.data);
+    })
+    .catch((err) => {
+      return Promise.resolve(err.response.data);
+    });
+}
```
Comment (dgoldstein1) [inline frontend/src/api/addProperty.js:18]:
```suggestion
```
we already have this in `src/api/search.js`

File: frontend/src/api/search.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import apiEndpoint from "./endpoint";
  **/
 
 export function postQuery(query) {
+  console.log("search query", query)
```
Comment (dgoldstein1) [inline frontend/src/api/search.js:9]:
```suggestion
```
remove logging

File: frontend/src/components/PropertyForm.js [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
+import HouseImage from "../images/house.svg"
+import BedImage from "../images/bed.svg"
+import BathImage from "../images/bath.svg"
+import SchoolImage from "../images/school.svg"
+import TicketImage from "../images/ticket.svg"
+import PersonImage from "../images/person.svg"
+import PhoneImage from "../images/phone.svg"
+import LocationImage from "../images/location.png"
+import BusImage from "../images/bus.svg"
+import BasementImage from "../images/basement.svg"
```
Comment (dgoldstein1) [inline frontend/src/components/PropertyForm.js:16]: add an index.js file to the images directory. Then do:
```suggestion
import * as images from "../images"
```
and access like so:
```js
images.BusImage
```

Comment (billyhunt) [reply frontend/src/components/PropertyForm.js:16]: I implemented an index file, but I only imported the ones we were using.  Let me know if this is ok.

**Pull request merged**